examples/get_iss_breast_data.py

```diff
@@ -0,0 +1,139 @@
+import argparse
+import io
+import os
+from typing import IO, Tuple
+
+from skimage.io import imread, imsave
+from slicedimage import ImageFormat
+
+from starfish.experiment.builder import FetchedTile, TileFetcher, write_experiment_json
+from starfish.types import Indices
+from starfish.util.argparse import FsExistsType
+
+SHAPE = 1044, 1390
+
+
+class IssCroppedBreastTile(FetchedTile):
+    def __init__(self, file_path):
+        self.file_path = file_path
+
+    @property
+    def shape(self) -> Tuple[int, ...]:
+        return SHAPE
+
+    @property
+    def format(self) -> ImageFormat:
+        return ImageFormat.TIFF
+
+    @staticmethod
+    def crop(img):
+        crp = img[40:1084, 20:1410]
+        return crp
+
+    def tile_data_handle(self) -> IO:
+        im = self.crop(imread(self.file_path))
+        fh = io.BytesIO()
+        imsave(fh, im, plugin='tifffile')
+        fh.seek(0)
+        return fh
+
+
+class ISSCroppedBreastPrimaryTileFetcher(TileFetcher):
+    def __init__(self, input_dir):
+        self.input_dir = input_dir
+
+    @property
+    def ch_dict(self):
+        ch_str = ['Cy3 5', 'Cy3', 'Cy5', 'FITC']
```

Comment:
not sure why this is better than just a literal dict.

Reply:
who cares.